ucsb-cs156-f24/team02-f24-10
# HelpRequests - CreatePage

Closes #49 

[Storybook Link](https://6722c06126c430949f67f157-tuauaceavb.chromatic.com/?path=/docs/pages-helprequest-helprequestcreatepage--docs)

Test Instructions:

- navigate to /helprequests/create 
![image](https://github.com/user-attachments/assets/c97adc2f-6eda-4fc1-9afa-312411bac073)

- Submit incorrectly filled form (to see errors)
![image](https://github.com/user-attachments/assets/ffce7745-fd3d-4ac0-9d5b-90364be5d47e)


- Fill out form correctly and submit
- Check the swagger to see if the request has been saved correctly
